pages/common/mmdc.md

```diff
@@ -0,0 +1,16 @@
+# mermaid-cli
+
+> A command-line interface (CLI) for mermaid. It takes a mermaid definition file as input and generates svg/png/pdf file as output.
```

Comment:
Perhaps this will read better?
```suggestion
> CLI for mermaid, a diagram generation tool with a domain-specific language.
> A mermaid definition file is taken as input and a svg, png, or pdf file is generated as output.
```